libraries/AP_BattMonitor/AP_BattMonitor_SMBus_NeoDesign.cpp

```diff
@@ -0,0 +1,102 @@
+#include <AP_HAL/AP_HAL.h>
+#include <AP_Common/AP_Common.h>
+#include <AP_Math/AP_Math.h>
```

Comment:
I'm sure it's incidentally included in some other header before here, but why do you need Math? I don't see anything  from it being used... (I only glanced quickly)